deployment/settings/services/imagetag.setting

```diff
@@ -1,4 +1,4 @@
 MINDARO_DEVHOSTAGENT_TAG=1.3.4
-MINDARO_DEVHOSTAGENT_RESTORATIONJOB_TAG=1.3.4
+MINDARO_DEVHOSTAGENT_RESTORATIONJOB_TAG=issue337
```

Comment:
this is temporary changes for testing , I will revert them back to right versions.